.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release pipeline
+
+on:
+  release:
+    types:
+      - released
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  INTEGRATION: "mysql"
+  ORIGINAL_REPO_NAME: 'newrelic/nri-mysql'
+  TAG: ${{ github.event.release.tag_name }}
+
+jobs:
+
+  publish-to-s3:
+    name: Send release assets to S3
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@adias-debug
+        env:
+          WORKDIR: /home/gha
+        with:
+          # lock enabled
+          disable_lock: false
+          run_id: ${{ github.run_id }}
+          env: release
+          tag: ${{env.TAG}}
+          app_name: "nri-${{env.INTEGRATION}}"
+          repo_name: ${{ env.ORIGINAL_REPO_NAME }}
+          # 'ohi' is for integrations
+          schema: "ohi"
+          aws_region: "us-east-1"
+          aws_role_arn: "arn:aws:iam::801306408012:role/s3-publisher"
```

Comment:
should this be a org secret shared among all repos sharing the action to have it centralized and be able to change it just in one place?

Reply:
This role is a temporary role I use for testing. We'll delete that. For production I'm not sure if it will be a org secret or we'll all use the same fixed role.

Reply:
caos said this will be a shared secret